tests/bluetooth/host/keys/bt_keys_get_addr/src/test_suite_full_list_invalid_values.c

```diff
@@ -22,8 +22,6 @@ extern struct bt_keys *returned_keys_refs[CONFIG_BT_MAX_PAIRED];
 
 static int bt_unpair_unreachable_custom_fake(uint8_t id, const bt_addr_le_t *addr)
 {
-	ARG_UNUSED(id);
-	ARG_UNUSED(addr);
```

Comment:
?

Reply:
This is a fake function callback to be used with unit test, these statements were there for testing and they aren't needed anymore.

Reply:
the `ARG_UNUSED` is there to note that these function arguments are intentionally unused and to prevent compiler warnings.

I do not see how that has anything to do with the FFF.

Reply:
Ok, you have a point here.

Reply:
I reverted that part, you can re-check